cfy_manager/main.py

```diff
@@ -224,18 +187,37 @@ def _validate_manager_installed(cmd):
             "Operation '{0}' is not allowed on a cluster node".format(cmd))
 
 
-def _get_components(with_attribute=None):
-    target_components = []
-    for component_name, component in COMPONENTS:
-        component_config = config.get(component_name, {})
-        if component_config.get('skip_installation'):
-            # This component is not installed, skip it
-            continue
-        if with_attribute and not hasattr(component, with_attribute):
-            # Desired attribute was not on this component
-            continue
-        target_components.append(component)
-    return target_components
+def _get_services_to_install():
+    """
+    Match all available services to install with all desired ones and
+    return only a unique list in case a services was asked for more then once
```

Comment:
unnecessary "in case a services was asked for more then once"

Reply:
Fixed